include/ignition/rendering/base/BaseAxisVisual.hh

```diff
@@ -78,6 +81,18 @@ namespace ignition
                                              _scale.Z());
     }
 
+    //////////////////////////////////////////////////
+    template <class T>
+    void BaseAxisVisual<T>::ShowAxisHead(bool b)
+    {
+      for (unsigned int i = 0; i < this->ChildCount(); ++i) {
```

Comment:
style nitpick, `{` on new line

there is also a minor codecheck error here:
https://build.osrfoundation.org/job/ignition_rendering-ci-pr_any-ubuntu_auto-amd64/629/cppcheckResult/